dependencies.yaml

```diff
@@ -152,13 +154,15 @@ files:
     includes:
       - common_build
       - depends_on_librmm
+      - depends_on_libcuvs
```

Comment:
Wheel builds shouldn't depend on libcuvs libraries, it should be fetched and built statically by CMake.
```suggestion
```